sources/models/shared-defaults/nvidia-k8s-container-toolkit.toml

```diff
@@ -0,0 +1,14 @@
+[settings.kubernetes.nvidia.container-runtime]
+visible-devices-as-volume-mounts = false
+visible-devices-envvar-when-unprivileged = true
```

Comment:
Lets use the default values to prevent unprivileged pods from accessing all the devices:

```toml
accept-nvidia-visible-devices-envvar-when-unprivileged = false
```